alaoweb/2021.alaoweb.org
# show both session start and Q&A time in modals

* closes #89 
* closes #185
* adds both session time and Q&A time next to modals
* to test
  * check out branch and set the config showSessions to `true`
  * then `bundle exec jekyll build`, `bundle exec jekyll serve`
  * You should see something like this for sessions with Q&A (most regular conference sessions):
<img width="701" alt="Screen Shot 2021-10-12 at 12 34 15 PM" src="https://user-images.githubusercontent.com/1894561/136995314-bf02e20e-1339-474b-93ec-a28a1cdd1979.png">
